intel-sgx/ias/src/client.rs

```diff
@@ -355,30 +331,31 @@ impl Client {
         // serde JSON deserialize, but binary data is Base64-encoded strings
         let mut deser = serde_json::Deserializer::from_slice(&raw_report);
         let deser = serde_bytes_repr::ByteFmtDeserializer::new_base64(&mut deser, base64::Config::new(base64::CharacterSet::Standard, true));
-        let report = VerifyAttestationEvidenceResponse::deserialize(deser).map_err(|e| format!("Error deserializing JSON response: {}", e))?;
+        let report: VerifyAttestationEvidenceResponse<Unverified> = VerifyAttestationEvidenceResponse::deserialize(deser)
+            .map_err(|e| format!("Error deserializing JSON response: {}", e))?;
+
+        debug!("IAS verification report: {:?}", report);
 
-        // Prior to API documentation v6.0, advisory_url and advisory_ids
-        // were returned in headers instead of the report itself
-        match (&advisory_url, &report.advisory_url) {
+        // since API documentation v6.0 advisory_url and advisory_ids
+        // are returned in the body of the response
```

Comment:
Is there a good reason these were still in `IasResult`? If not, we can remove them and there is also no longer a need to parse the `VerifyAttestationEvidenceResponse` in this function

Reply:
These are here to be able to offer a consistent Rust API regardless of REST API version?

Reply:
If this crate was already publicly available before v6.0, I could understand but this is the initial public version. Do we expect IAS to go back to a previous version? Or are we using a proxy that may not use v6.0?